### PR TITLE
VAR-179 | Replace failure message and add a return button

### PR DESCRIPTION
**Pull request by Nizar-Rahme**

### CODE REVIEW

File: app/i18n/messages/sv.json [hunk 1]
```diff
@@ -48,7 +48,8 @@
   "common.totalPriceLabel": "Total pris",
   "common.priceWithVAT": "{price}€ (inkl. moms {vat}%)",
   "payment.title": "Betalning avbruten",
-  "payment.text": "Prova igen om 15 minuter",
+  "payment.text": "Betalning avbruten.",
+  "payment.return": "Lämna tillbaka",
```
Comment (badrange) [inline app/i18n/messages/sv.json]: Where did you get this translation from?

Comment (Nizar-Rahme) [reply app/i18n/messages/sv.json]: From @SamuelSaarentola  https://helsinkisolutionoffice.atlassian.net/browse/VAR-179

Comment (badrange) [reply app/i18n/messages/sv.json]: @SamuelSaarentola I'm not a native Swedish speaker, so I'm not going to argue about this, I just wanted to mention that I read "lämnä tillbaka" as "return" in the same way as you'll return a book to the library. Just FYI :-)

Comment (SamuelSaarentola) [reply app/i18n/messages/sv.json]: @badrange @Nizar-Rahme Im not either, more correct would be just **tillbaka**

File: app/pages/reservation/PaymentFailed.js [hunk 2]
```diff
@@ -7,12 +7,18 @@ import injectT from '../../i18n/injectT';
...
   t,
+  resourceId
 }) {
   return (
     <div className="app-ReservationFailedPage__contentbox">
       <h2>{t('payment.title')}</h2>
       <p>{t('payment.text')}</p>
-      <Link className="app-ReservationFailedPage__link" to="/my-reservations">
+      {resourceId && (
+        <Link className="app-ReservationFailedPage__link" to={`/resources/${resourceId}`}>
```
Comment (midori0507) [inline app/pages/reservation/PaymentFailed.js]: Should `history.goBack()` is better (so all queries was preserved)

Comment (Nizar-Rahme) [reply app/pages/reservation/PaymentFailed.js]: I thought about it but, the previous page is the Bambora form, and using `goBack()` prevents you from opening this URL in a new tab.

Comment (midori0507) [reply app/pages/reservation/PaymentFailed.js]: Hah, true!

**Pull request merged**